src/emulate.c

```diff
@@ -644,7 +661,7 @@ static void block_translate(riscv_t *rv, block_t *block)
             break;
         }
 
-        ir = mpool_calloc(rv->block_ir_mp);
+        ir = mpool_alloc(rv->block_ir_mp);
```

Comment:
Ditto.

Reply:
It seems like a rebase problem for #437.